src/main/java/com/igeekinc/util/macos/macosx/MacOSXFilePath.java

```diff
@@ -59,6 +58,7 @@ private MacOSXFilePath()
 	/* (non-Javadoc)
 	 * @see com.igeekinc.util.FilePath#getNewFilePath(java.lang.String[], int, int, boolean)
 	 */
+ @Override
```

Comment:
Please check with indent here. Same on line 74

Reply:
It will be fixed in new PR.